lib/src/signed_video_h26x_sign.c

```diff
@@ -158,22 +159,22 @@ complete_sei_nalu_and_add_to_prepend(signed_video_t *self)
   SVI_DONE(status)
 
 done:
-  // Done with the SEI payload. Move |sei_data_buffer|. This should be done even if we caught a
-  // failure.
+
+  return status;
+}
+void
+free_and_reset_sei_data_buffer(signed_video_t *self)
```

Comment:
The function name does not really reflect what it is doing. When I read the code it removes the first element. There is no free nor reset involved.